azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/Utils.java

```diff
@@ -82,7 +82,12 @@ public static void copyResources(final MavenProject project, final MavenSession
                                      final MavenResourcesFiltering filtering, final List<Resource> resources,
                                      final String targetDirectory) throws IOException {
         for (final Resource resource : resources) {
-            resource.setTargetPath(Paths.get(targetDirectory, resource.getTargetPath()).toString());
+            final String targetPath = resource.getTargetPath();

```

Comment:
Can we make it simpler?
```java
final String targetPath = resource.getTargetPath() == null ? "" : resource.getTargetPath();
resource.setTargetPath(Paths.get(targetDirectory, targetPath).toString());
```

Reply:
sure, but if targetPath is empty string, my change could save a call to Paths.get, that's why I use if block